bazel_example/diregapic/BUILD.bazel

```diff
@@ -53,15 +53,19 @@ ruby_gapic_assembly_pkg(
 ruby_cloud_gapic_library(
   name = "compute_ruby_gapic",
   srcs = [":compute_proto_with_info"],
-  ruby_cloud_title="Compute V1",
-  ruby_cloud_description="google-cloud-compute-v1 is the official client library for the Google Cloud Compute V1 API.",
   extra_protoc_parameters = [
+    "ruby-cloud-api-id=compute.googleapis.com",
+    "ruby-cloud-api-shortname=compute",
     "ruby-cloud-gem-name=google-cloud-compute-v1",
-    "ruby-cloud-generate-transports=rest",
     "ruby-cloud-generate-metadata=false",
-    "ruby-cloud-extra-dependencies=gapic-common=>= 0.4.1|< 2.a",
+    "ruby-cloud-generate-transports=rest",
+    "ruby-cloud-env-prefix=COMPUTE",
+    "ruby-cloud-extra-dependencies=gapic-common=>= 0.5.1|< 2.a",
```

Comment:
Will this work? I was under the impression that we couldn't include spaces in these values, and that ruby_cloud_description and ruby_cloud_title were separate fields because of this limitation. The gem requirements syntax should be able to handle no space between the operator and the version (`>=0.5.1|<2.a`)

Reply:
somehow that bazel bug does not touch this particular line. I'll go in and figure out why after the regapic rush, but it is working right now since this is how I generated a client to acceptance-test